python/aitemplate/backend/cuda/target_def.py

```diff
@@ -136,10 +136,15 @@ def get_host_compiler_options(self) -> List[str]:
         return self._build_gnu_host_compiler_options()
 
     def _build_nvcc_compiler_options(self) -> List[str]:
+        code = [f"sm_{self._arch}", f"compute_{self._arch}"]
+        code = ",".join(
+            code + [f"lto_{self._arch}"] if environ.enable_cuda_lto() else []
```

Comment:
`enable_cuda_lto` is different from `enable_cuda_lto_target` in `environ.py`. I'd rather rename it to `enable_cuda_lto` in `environ.py`, to better align with the environment variable name.